### PR TITLE
catching key errors for case properties

### PR DESCRIPTION
**Pull request by calellowitz**
@NoahCarnahan 
cc: @emord @kaapstorm 
Fixes http://manage.dimagi.com/default.asp?218852#1099524
It was 500ing when a deleted case property was set as the aggregation column for a report.

### CODE REVIEW

File: corehq/apps/userreports/reports/builder/forms.py [hunk 1]
```diff
@@ -802,7 +802,9 @@ def _get_view_model(self, filter):
         )
 
     def _get_property_from_column(self, col):
-        return self._properties_by_column[col].id
+        column =  self._properties_by_column.get(col)
```
Comment (dimagimon) [inline corehq/apps/userreports/reports/builder/forms.py]: E222 multiple spaces after operator

**Comment by NoahCarnahan** [comment]
[This](https://github.com/dimagi/commcare-hq/blob/66dcf1e9f7f255f286c62b142a253451569d33aa/corehq/apps/userreports/reports/builder/forms.py#L916) is the call to `_get_property_from_column()` that fails, right?
This is a nitpick, but it might be easier to see what is going on if we didn't change `_get_property_from_column()`, and called `self._column_exists(existing_agg_cols[0])` at the linked location instead. I don't feel strongly about this though.

**Comment by calellowitz** [comment]
Yeah, it was that.
I did it that way because it looked like other calls to `_get_property_from_column` may run into similar issues. notably https://github.com/dimagi/commcare-hq/blob/66dcf1e9f7f255f286c62b142a253451569d33aa/corehq/apps/userreports/reports/builder/forms.py#L1145 and https://github.com/dimagi/commcare-hq/blob/66dcf1e9f7f255f286c62b142a253451569d33aa/corehq/apps/userreports/reports/builder/forms.py#L1221 could break if the relevant properties were deleted. seemed easier to catch them all in one place, but happy to change if you think this could cover up other issues

**Comment by NoahCarnahan** [comment]
:+1:

**Pull request merged**